docs/source/getting_started/basic_usage.rst

```diff
@@ -0,0 +1,115 @@
+Basic project commands
```

Comment:
If I've done this before, and have just forgotten what the third command was, I don't want to have to read this whole page to find it.
Maybe a quick refresh section at the very start for returning authors. Anyone else can read on to understand.